### PR TITLE
Add LCD graphics

### PR DESCRIPTION
**Pull request by 0x050f**

### CODE REVIEW

File: .github/workflows/ci.yml
Comment (louisabricot) [inline .github/workflows/ci.yml:1]: Not sure that the gameboy font is very practical for the debugger, maybe look for a clearer font ?

**Review by louisabricot — CHANGES_REQUESTED** [review]
Replace the hardcoded values and let's merge !  It looks great !
Also, don't be shy with documentation 🧙‍♀️

File: src/graphics/lcd.rs [hunk 1]
```diff
@@ -0,0 +1,259 @@
...
+            .find(|button| button.rect().contains_point(Point::new(x, y)))
+    }
+
+    /// Print the actual frame into the LCD window
+    pub fn print_frame(&mut self) {
+        self.canvas.clear();
+        let mut x: u32 = 0;
+        let mut y: u32 = 0;
+        while y < self.get_height() {
+            let c1: u8 = (((x * 255) / self.get_width()) % 255) as u8;
```
[the remark this answers is not in the document]
Comment (0x050f) [reply src/graphics/lcd.rs:206]: Will be replaced by GameBoy rendering

File: .github/workflows/ci.yml
Comment (0x050f) [reply .github/workflows/ci.yml:1]: Actually the gameboy font isn't mandatory for dev -> Issued

**Review by louisabricot — APPROVED** [review]
PERFECT !!

**Pull request merged**